plugins/verifyRequest.js

```diff
@@ -1,15 +1,11 @@
 import createError from 'http-errors'
 import { createVerificationSignature } from '../helpers/crypto.js'
 
-export default async function verifyRequest(pluginOptions, req) {
-  const { clientid } = req.headers
+export default async function verifyRequest(req) {
   const timestamp = req.headers['x-zm-request-timestamp']
   const signature = createVerificationSignature(timestamp, req.body)
 
-  if (
-    clientid !== pluginOptions.clientId ||
```

Comment:
Maybe I am missing some context, but why are you removing the client.id check?

Reply:
They stopped sending the client id header. You can check in their documentation that in the Node.js example they do not check the client id: https://developers.zoom.us/docs/api/rest/webhook-reference/#verify-with-zooms-header

Reply:
@RomuloVitoi if they're not sending it anymore, can you explain why this check kept working?